src/contracts/middleware/VoteWeigherBaseStorage.sol

```diff
@@ -44,34 +35,33 @@ abstract contract VoteWeigherBaseStorage is Initializable, IVoteWeigher {
     /// @notice The ServiceManager contract for this middleware, where tasks are created / initiated.
     IServiceManager public immutable serviceManager;
 
-    /// @notice Number of quorums that are being used by the middleware.
-    uint256 public immutable NUMBER_OF_QUORUMS;
+    /// @notice The number of quorums that the VoteWeigher is considering.
+    uint16 public quorumCount;
 
     /**
      * @notice mapping from quorum number to the list of strategies considered and their
      * corresponding multipliers for that specific quorum
      */
-    mapping(uint256 => StrategyAndWeightingMultiplier[]) public strategiesConsideredAndMultipliers;
+    mapping(uint8 => StrategyAndWeightingMultiplier[]) public strategiesConsideredAndMultipliers;
 
     /**
      * @notice This defines the earnings split between different quorums. Mapping is quorumNumber => BIPS which the quorum earns, out of the total earnings.
      * @dev The sum of all entries, i.e. sum(quorumBips[0] through quorumBips[NUMBER_OF_QUORUMS - 1]) should *always* be 10,000!
      */
-    mapping(uint256 => uint256) public quorumBips;
+    mapping(uint8 => uint256) public quorumBips;
```

Comment:
looks like we aren't using nor setting this right now.
that's fine, but the associated comment is definitely a bit of a scare here.
perhaps just something to track as a TODO going forward?

Reply:
conclusion: remove this storage